scripts/zones/Aht_Urhgan_Whitegate/npcs/Lathuya.lua

```diff
@@ -2,17 +2,50 @@
 -- Area: Aht Urhgan Whitegate
 --  NPC: Lathuya
 -- Standard Info NPC
+-- Involved in quests: Omens (BLU AF2)
+-- !pos -95.081 -6.000 31.638 50
+-----------------------------------
+require("scripts/globals/quests")
+require("scripts/globals/npc_util")
 -----------------------------------
 
 function onTrade(player,npc,trade)
 end
 
 function onTrigger(player,npc)
-    player:startEvent(770)
+    -- OMENS
+    if player:hasKeyItem(tpz.ki.SEALED_IMMORTAL_ENVELOPE) then
```

Comment:
Just for readability of the code could you add local getqueststatus(..., omens) and then add that to the check with has envelope.

Reply:
For sure but the check for envelope is only being used once. From old PRs I have submitted on the other project that shall not be named, I was told to do that only if it is being used more than once for readability :)

Tagging @ibm2431  for second opinion.

Reply:
Just the readability part. Like just looking at the code I wouldn't know it's for this quest because the check is just an envelope ki that I don't know where it comes from

Reply:
Fixed. Will be pushing change soon.